res/database/Default/database_ads.xml

```diff
@@ -194,7 +195,7 @@ You will really have to chew.</en>
 			</title>
 			<description>
 				<de>Action-Spielfiguren Deiner Lieblingspolitiker. Für kurze Zeit Birne und Genschman zum Preis von einem.</de>
-				<en>Action game figures of your favorite politicians. For a short time Birne and Genschman for the price of one.</en>
+				<en>Action play figures of your favorite politicians. For a short time Pear and Genschman for the price of one.</en>
```

Comment:
Not sure if you want to translate the moniker "Birne" there - depends on the nickname they gave "him" outside of Germany.

Reply:
'They called him “The Pear.”'

https://www.moderntreatise.com/international/tag/Helmut+Kohl

"The Pear" or "Pearhead" might be even better.

Reply:
OK, then Pear it is.